### ISSUE TITLE
[Accessibility- Script C 7.2] News_details- Some buttons don't contain a title

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3000]
**Environment:** Tribe

**Current behavior:**

Some buttons in the article have no labels or non explicit one:

- The button that displays the list of kudos

- The button that displays the list of likes on comments

- The “See more” button to see the list of likes on the article

**Expected behavior:**

To add an explicit aria-label ou title to buttons cited above
Titles must respect the rule “the visible name must be included (at least) in the accessible name”.

**Debate topic**

Which attribute must be used as a title for buttons (title or aria-label)? 

![Image](https://github.com/user-attachments/assets/fa102a3d-0b07-45ef-b687-7450f8e1119f)

**Comment by mbensalem** [comment]
Still reproduced on 7.2.x
